shell/plugins/dashboard-store/__tests__/actions.spec.js

```diff
@@ -0,0 +1,125 @@
+import _actions from '@shell/plugins/dashboard-store/actions';
+
```

Comment:
We have been always using TS for the tests, could we stick to it also for this case?
If it's too complicated, could you please list the difficulties you crossed so we can tackle them down another time?

Reply:
There was a type exception showing on my linter on assertions that seemed to be the result of a conflict with the assertions from cypress. Upon further investigation, it looks like we see that conflict in all existing tests as well and we're okay with that so I changed my tests over to TS.